src/utils/getIsInputEmpty.js

```diff
@@ -0,0 +1,13 @@
+function getIsInputEmpty () {
+  const items = Object.values(arguments);
+  for (let i = 0; i < items.length; i++) {
+    if (items[i] === undefined) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
```suggestion
items.map((item) => return !item)
```
같은 동작을 수행하는지 확인 부탁드립니다 :)

Reply:
서로 다르게 동작합니다.
인풋 중에 하나라도 falsy한 값이면 바로 true로 반환하게 만든 함수입니다